test_values/test_cases.json

```diff
@@ -252,5 +252,196 @@
       ],
       "expected": true
     }
+
```

Comment:
Could we order the fields in the same order? Some cases have `roots` before `proof_hashes` but some don't. It'd be easier to read if they were in the same order.

Reply:
The ordering is still mixed. I see that most of the test cases have the ordering of:

leaf_values, targets, roots, proof

but there's still many that don't follow the above ordering